miniapps/nurbs/nurbs_int.cpp

```diff
@@ -0,0 +1,164 @@
+//                                MFEM Example interpolation -- modified for NURBS FE
+//
+// Compile with: make nurbs_int
+//
+// Sample runs:  nurbs_int -m ../../data/pipe-nurbs-2d.mesh -o 2
+
+
+//
+// Description:  This example code illustrates usage of mixed finite element
+//               spaces, with three variants:
+//               0) H1
+//               1) H(div)
+//               2) H(curl)
+
+
+#include "mfem.hpp"
+#include <fstream>
+#include <iostream>
+
+using namespace std;
+using namespace mfem;
+
+void vfun(const Vector &x, Vector &v)
+{
+   v(0) = sin(6.28 * x(0));
+   v(1) = sin(6.28 * x(1));
+   if (x.Size() == 3) { v(2) = sin(6.28 * x(2)); }
+}
+
+
+int dim;
+real_t freq = 1.0, kappa;
+
+int main(int argc, char *argv[])
+{
+   // 1. Parse command-line options.
+   int dim = 2;
+   int ref_levels = 0;
+   int order = 1;
+   bool NURBS = true;
+   int prob = 0;
+   bool static_cond = false;
```

Comment:
Unused: `static_cond`, `pa`, `visualization`.

Reply:
The `nurbs_int` was meanly a debug vehicle. There is no real need for the app, so I decided to remove it.